api/db/changelog/db.changelog-master.xml

```diff
@@ -61,4 +61,5 @@
   <include file="changelog/db.changelog-51-user-profile-updates.xml"/>
   <include file="changelog/db.changelog-52-user-nonce.xml"/>
   <include file="changelog/db.changelog-53-research-purpose-type.xml"/>
+  <include file="changelog/db.changelog-54-moodle-id-to-user.xml"/>
```

Comment:
Note that this doesn't match the actual filename `db.changelog-54-add-moodle-id-to-user.xml` which is why the test is failing.